pangeo_forge_runner/storage.py

```diff
@@ -67,17 +75,20 @@ def get_forge_target(self, job_name: str):
 
         cls = getattr(storage, self.pangeo_forge_target_class)
 
+
         # pangeo-forge-recipes >=0.10.5 have a new `fsspec_kwargs` kwarg
         if any(field.name == "fsspec_kwargs" for field in fields(cls)):
             return cls(
                 self.fsspec_class(**self.fsspec_args),
                 root_path=self.root_path.format(job_name=job_name),
                 fsspec_kwargs=self.fsspec_args,
+                **self.pangeo_forge_target_class_args,
```

Comment:
Might be good to disambiguate the intents of `pangeo_forge_target_class_args` and `fsspec_args`